### PR TITLE
docs: update README

### PR DESCRIPTION
**Pull request by joshuajames-smith**
## Description

Update documentation to this repo's README which has absorbed and expanded upon the previous version to help improve the gliff.ai GitHub presence and consistency.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,98 @@
...
-A preview build of the current `main` branch is [here](https://annotate.staging.gliff.app/)
+ANNOTATE aims to allow domain experts to easily and efficiently annotate multidimensional images - embedding their knowledge - for the purposes of developing imaging AI products. ANNOTATE is just one part of [gliff.ai](https://gliff.ai/)’s growing privacy-preserving MLOps (Machine Learning Operations) platform. When the full [gliff.ai platform](https://gliff.ai/software/) is used, ANNOTATE provides just one step in developing high-quality and auditable datasets that satisfy any relevant regulatory frameworks which enables our users to build world-changing and trustworthy AI models and products.
+
+A preview build of the current `main` branch is [here](https://annotate.staging.gliff.app/).
+
+✅ **We welcome contributions on this repository!** ✅
+
+## Demo-Preview
+
+_Coming soon._
```
Comment (ChasNelson1990) [inline README.md:17]: There is a staging demo at: https://annotate.staging.gliff.app/
@cooper667 shall we just link to this (same for other repos too)?

File: README.md [hunk 2]
```diff
@@ -1,46 +1,98 @@
...
+
+A big thank you from the entire gliff.ai team to these fellow contributors ([emoji key](https://allcontributors.org/docs/en/emoji-key)): \
+[{{Contributor List - _coming soon_}}](https://github.com/all-contributors/all-contributors)
+
+## Contact
+
+[{{back to navigation}}](#table-of-contents)
+
+Need some help? 🤔 Have a question? 🧠 \
+Reach out to the gliff.ai team at [community@gliff.ai](mailto:community@gliff.ai?subject=[GitHub]) or on GitHub.
```
Comment (ChasNelson1990) [inline README.md:91]: What about issues? And do we want to use Github Discussion boards?

File: README.md (hunk 1 above)
Comment (cooper667) [reply README.md:17]: yea, i think so, just explain it's built from the /examples folder on `main`? same for almost every repo

File: README.md (hunk 2 above)
Comment (joshuajames-smith) [reply README.md:87]: @ChasNelson1990 I think having a Github Discussion board for each public repo is a great idea! Can you switch it on for this repo as an initial test?

Comment (ChasNelson1990) [reply README.md:87]: Done

**Review by cooper667 — APPROVED** [review]

**Review by ChasNelson1990 — APPROVED** [review]

**Pull request merged**